src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/jpa/repository/TierCalculationRepositoryTest.kt

```diff
@@ -57,8 +58,55 @@ class TierCalculationRepositoryTest(
     fun `Should return latest calculation when multiple`() {
 
       val created = LocalDateTime.now()
-      val firstTierCalculation = TierCalculationEntity(crn = crn, created = created, data = data)
-      val secondTierCalculation = TierCalculationEntity(crn = "$crn 2", created = created.minusSeconds(1), data = data)
+      val firstTierCalculation = TierCalculationEntity(crn = crn, created = created, data = data, uuid = UUID.randomUUID())
+      val secondTierCalculation = TierCalculationEntity(crn = "$crn 2", created = created.minusSeconds(1), data = data, uuid = UUID.randomUUID())
```

Comment:
I might be misreading this but it looks like the repository has calculations with different CRNs so findFirst will always return the one which matches the crn

Reply:
That's a really good point, I'll check the git blame. 🤡